src/pages/NIHicWebform.js

```diff
@@ -0,0 +1,2266 @@
+import {Component} from 'react';
+import {RadioButton} from '../components/RadioButton';
+import {a, div, fieldset, form, h, h3, hr, input, label, span, textarea} from 'react-hyperscript-helpers';
+import Select from 'react-select';
+import {Notification} from '../components/Notification';
+import {PageHeading} from '../components/PageHeading';
+import {DAR} from '../libs/ajax';
+import {searchOntology} from '../libs/ontologyService';
+import * as fp from 'lodash/fp';
+import AsyncSelect from 'react-select/async';
+
+class NIHICWebform extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      dacList: [],
+      selectedDac: {},
+      nihValid: false,
+      showValidationMessages: false,
+      formData: {
+        methods: '',
+        genetic: '',
+        publication: '',
+        collaboration: '',
+        ethics: '',
+        geographic: '',
+        moratorium: '',
+        populationMigration: '',
+        forProfit: '',
+        hmb: false,
+        poa: false,
+        diseases: false,
+        ontologies: [],
+        other: false,
+        otherText: '',
+        generalUse: false
+      },
+      datasetData: {
```

Comment:
Once you remove all references to `datasetData`, this state variable can go away.